<problems>
<problem>
  <file>file://$PROJECT_DIR$/cogs/onboarding.py</file>
  <line>114</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/onboarding.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'gray' for class 'Colour'</description>
  <highlighted_element>gray</highlighted_element>
  <language>Python</language>
  <offset>76</offset>
  <length>4</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/onboarding.py</file>
  <line>130</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/onboarding.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'processed_at' for class 'OnboardingSession'</description>
  <highlighted_element>processed_at</highlighted_element>
  <language>Python</language>
  <offset>30</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/onboarding.py</file>
  <line>133</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/onboarding.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'processed_at' for class 'OnboardingSession'</description>
  <highlighted_element>processed_at</highlighted_element>
  <language>Python</language>
  <offset>65</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>77</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'option_index' for class 'PollVote'</description>
  <highlighted_element>option_index</highlighted_element>
  <language>Python</language>
  <offset>32</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>79</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'option_index' for class 'PollVote'</description>
  <highlighted_element>option_index</highlighted_element>
  <language>Python</language>
  <offset>35</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>85</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'is_anonymous' for class 'Poll'</description>
  <highlighted_element>is_anonymous</highlighted_element>
  <language>Python</language>
  <offset>24</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>87</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'option_index' for class 'PollVote'</description>
  <highlighted_element>option_index</highlighted_element>
  <language>Python</language>
  <offset>36</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>98</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'question' for class 'Poll'</description>
  <highlighted_element>question</highlighted_element>
  <language>Python</language>
  <offset>43</offset>
  <length>8</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>118</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'is_anonymous' for class 'Poll'</description>
  <highlighted_element>is_anonymous</highlighted_element>
  <language>Python</language>
  <offset>28</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>136</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'is_anonymous' for class 'Poll'</description>
  <highlighted_element>is_anonymous</highlighted_element>
  <language>Python</language>
  <offset>62</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>137</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'status' for class 'Poll'</description>
  <highlighted_element>status</highlighted_element>
  <language>Python</language>
  <offset>50</offset>
  <length>6</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>139</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'closes_at' for class 'Poll'</description>
  <highlighted_element>closes_at</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>140</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'closes_at' for class 'Poll'</description>
  <highlighted_element>closes_at</highlighted_element>
  <language>Python</language>
  <offset>58</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>141</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'closes_at' for class 'Poll'</description>
  <highlighted_element>closes_at</highlighted_element>
  <language>Python</language>
  <offset>82</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>143</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'closes_at' for class 'Poll'</description>
  <highlighted_element>closes_at</highlighted_element>
  <language>Python</language>
  <offset>82</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>145</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'author_id' for class 'Poll'</description>
  <highlighted_element>author_id</highlighted_element>
  <language>Python</language>
  <offset>51</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>209</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'question' for class 'Poll'</description>
  <highlighted_element>question</highlighted_element>
  <language>Python</language>
  <offset>56</offset>
  <length>8</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>210</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>'Embed' object has no attribute 'color'</description>
  <highlighted_element>color</highlighted_element>
  <language>Python</language>
  <offset>35</offset>
  <length>5</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>218</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'question' for class 'Poll'</description>
  <highlighted_element>question</highlighted_element>
  <language>Python</language>
  <offset>38</offset>
  <length>8</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>228</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'question' for class 'Poll'</description>
  <highlighted_element>question</highlighted_element>
  <language>Python</language>
  <offset>33</offset>
  <length>8</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>240</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'author_id' for class 'Poll'</description>
  <highlighted_element>author_id</highlighted_element>
  <language>Python</language>
  <offset>29</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>263</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'status' for class 'Poll'</description>
  <highlighted_element>status</highlighted_element>
  <language>Python</language>
  <offset>40</offset>
  <length>6</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>274</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'question' for class 'Poll'</description>
  <highlighted_element>question</highlighted_element>
  <language>Python</language>
  <offset>44</offset>
  <length>8</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>274</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'question' for class 'Poll'</description>
  <highlighted_element>question</highlighted_element>
  <language>Python</language>
  <offset>77</offset>
  <length>8</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>276</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'status' for class 'Poll'</description>
  <highlighted_element>status</highlighted_element>
  <language>Python</language>
  <offset>40</offset>
  <length>6</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/cogs/polls.py</file>
  <line>314</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/cogs/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'status' for class 'Poll'</description>
  <highlighted_element>status</highlighted_element>
  <language>Python</language>
  <offset>41</offset>
  <length>6</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/context_menus.py</file>
  <line>10</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/context_menus.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'ModerationLog' in 'database.py'</description>
  <highlighted_element>ModerationLog</highlighted_element>
  <language>Python</language>
  <offset>51</offset>
  <length>13</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/context_menus.py</file>
  <line>161</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/context_menus.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'UserRoleManagerView' in 'moderation.py'</description>
  <highlighted_element>UserRoleManagerView</highlighted_element>
  <language>Python</language>
  <offset>37</offset>
  <length>19</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/context_menus.py</file>
  <line>181</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/context_menus.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'discord_id' for class 'User'</description>
  <highlighted_element>discord_id</highlighted_element>
  <language>Python</language>
  <offset>40</offset>
  <length>10</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/context_menus.py</file>
  <line>209</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/context_menus.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'UserProfileView' in 'profiles.py'</description>
  <highlighted_element>UserProfileView</highlighted_element>
  <language>Python</language>
  <offset>35</offset>
  <length>15</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/database.py</file>
  <line>309</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/database.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'dispose' for class 'None'</description>
  <highlighted_element>dispose</highlighted_element>
  <language>Python</language>
  <offset>22</offset>
  <length>7</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/configuration.py</file>
  <line>1054</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/configuration.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'send' in 'AppCommandChannel | AppCommandThread'</description>
  <highlighted_element>send</highlighted_element>
  <language>Python</language>
  <offset>36</offset>
  <length>4</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/configuration.py</file>
  <line>1118</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/configuration.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'send' in 'AppCommandChannel | AppCommandThread'</description>
  <highlighted_element>send</highlighted_element>
  <language>Python</language>
  <offset>36</offset>
  <length>4</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/moderation.py</file>
  <line>1206</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/moderation.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'action_taken' for class 'ModerationIncident'</description>
  <highlighted_element>action_taken</highlighted_element>
  <language>Python</language>
  <offset>44</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/panels.py</file>
  <line>103</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/panels.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'PollManagerView' in 'polls.py'</description>
  <highlighted_element>PollManagerView</highlighted_element>
  <language>Python</language>
  <offset>32</offset>
  <length>15</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/panels.py</file>
  <line>163</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/panels.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'ProfileAdminView' in 'profiles.py'</description>
  <highlighted_element>ProfileAdminView</highlighted_element>
  <language>Python</language>
  <offset>35</offset>
  <length>16</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/panels.py</file>
  <line>321</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/panels.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="ERROR" attribute_key="ERRORS_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'ReportMessageModal' in 'moderation.py'</description>
  <highlighted_element>ReportMessageModal</highlighted_element>
  <language>Python</language>
  <offset>37</offset>
  <length>18</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>376</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'status' for class 'Poll'</description>
  <highlighted_element>status</highlighted_element>
  <language>Python</language>
  <offset>20</offset>
  <length>6</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>380</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'closes_at' for class 'Poll'</description>
  <highlighted_element>closes_at</highlighted_element>
  <language>Python</language>
  <offset>20</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>380</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'closes_at' for class 'Poll'</description>
  <highlighted_element>closes_at</highlighted_element>
  <language>Python</language>
  <offset>77</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>441</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'option_index' for class 'PollVote'</description>
  <highlighted_element>option_index</highlighted_element>
  <language>Python</language>
  <offset>32</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>443</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'option_index' for class 'PollVote'</description>
  <highlighted_element>option_index</highlighted_element>
  <language>Python</language>
  <offset>35</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>449</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'is_anonymous' for class 'Poll'</description>
  <highlighted_element>is_anonymous</highlighted_element>
  <language>Python</language>
  <offset>24</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>451</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'option_index' for class 'PollVote'</description>
  <highlighted_element>option_index</highlighted_element>
  <language>Python</language>
  <offset>36</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>462</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'question' for class 'Poll'</description>
  <highlighted_element>question</highlighted_element>
  <language>Python</language>
  <offset>43</offset>
  <length>8</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>480</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'is_anonymous' for class 'Poll'</description>
  <highlighted_element>is_anonymous</highlighted_element>
  <language>Python</language>
  <offset>28</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>498</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'is_anonymous' for class 'Poll'</description>
  <highlighted_element>is_anonymous</highlighted_element>
  <language>Python</language>
  <offset>62</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>500</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'closes_at' for class 'Poll'</description>
  <highlighted_element>closes_at</highlighted_element>
  <language>Python</language>
  <offset>16</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>501</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'closes_at' for class 'Poll'</description>
  <highlighted_element>closes_at</highlighted_element>
  <language>Python</language>
  <offset>58</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>502</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'closes_at' for class 'Poll'</description>
  <highlighted_element>closes_at</highlighted_element>
  <language>Python</language>
  <offset>82</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>506</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'author_id' for class 'Poll'</description>
  <highlighted_element>author_id</highlighted_element>
  <language>Python</language>
  <offset>51</offset>
  <length>9</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>558</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'question' for class 'Poll'</description>
  <highlighted_element>question</highlighted_element>
  <language>Python</language>
  <offset>33</offset>
  <length>8</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>575</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'option_index' for class 'PollVote'</description>
  <highlighted_element>option_index</highlighted_element>
  <language>Python</language>
  <offset>36</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>577</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Unresolved attribute reference 'option_index' for class 'PollVote'</description>
  <highlighted_element>option_index</highlighted_element>
  <language>Python</language>
  <offset>39</offset>
  <length>12</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>615</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'question' in 'Poll | None'</description>
  <highlighted_element>question</highlighted_element>
  <language>Python</language>
  <offset>43</offset>
  <length>8</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/polls.py</file>
  <line>616</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/polls.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>'Embed' object has no attribute 'color'</description>
  <highlighted_element>color</highlighted_element>
  <language>Python</language>
  <offset>22</offset>
  <length>5</length>
</problem>
<problem>
  <file>file://$PROJECT_DIR$/views/roles.py</file>
  <line>474</line>
  <module>guildbot</module>
  <entry_point TYPE="file" FQNAME="file://$PROJECT_DIR$/views/roles.py" />
  <problem_class id="PyUnresolvedReferencesInspection" severity="WARNING" attribute_key="WARNING_ATTRIBUTES">Unresolved references</problem_class>
  <description>Cannot find reference 'defer' in '() -&gt; InteractionResponse'</description>
  <highlighted_element>defer</highlighted_element>
  <language>Python</language>
  <offset>35</offset>
  <length>5</length>
</problem>
</problems>